src/refract/JsonValue.cc

```diff
@@ -284,6 +285,12 @@ namespace
         return renderValue(*merged, options);
     }
 
+    so::Value renderValueSpecific(const RefElement& element, TypeAttributes options)
```

Comment:
I wonder, could there be other cases where we would render a ref element outside an enum here? Trying to understand if there is perhaps some other test-coverage we should add for other types.

Reply:
Perhaps we should explore that. Also, minim has that `generateValue` function that probably manifests the same bug.